packages/api/db/tables.sql

```diff
@@ -0,0 +1,28 @@
+-- User tags are associated to a user for the purpose of granting/restricting them
+-- in the application.
+-- Needed per https://www.postgresql.org/message-id/20141203213139.GA9855%40fetter.org
```

Comment:
Postgres does not allow us to use this from fdw, so we have to also add it here. I believe that only work around would be to have nft.storage DB within a namespace instead of in public